app/src/forms/bcgeoaddress/service.js

```diff
@@ -2,7 +2,22 @@ const geoAddressService = require('../../components/geoAddressService');
 
 const service = {
   searchBCGeoAddress: async (query) => {
-    return geoAddressService.addressQuerySearch(query);
+    let addresses = { features: [] };
+
+    let searchAddresses = await geoAddressService.addressQuerySearch(query);
+
+    if (searchAddresses?.features && Array.isArray(searchAddresses.features)) {
+      await searchAddresses.features.forEach((element) => {
```

Comment:
@timisenco2015 why would we need `await` here?

Reply:
It was returning empty addresses

Reply:
Actually, it is not needed. I will remove it.